### PR TITLE
fix: remove cache from node endpoints

### PR DESCRIPTION
**Pull request by giovanecosta**

### CODE REVIEW

**Review by pedroferreira1 — APPROVED** [review]

File: serverless.yml [hunk 1]
```diff
@@ -78,10 +78,6 @@ functions:
             parameters: 
               paths: 
                 hash: true
-          caching:
```
Comment (msbrogli) [inline serverless.yml:81]: Why are you removing cache?

**Review by msbrogli — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (luislhl) [reply serverless.yml:81]: I'm also curious about this.

Comment (giovanecosta) [reply serverless.yml:81]: this is the objective of the PR

Comment (luislhl) [reply serverless.yml:81]: But why it had to be removed?

Comment (luislhl) [reply serverless.yml:81]: Ok, I have aligned this via Slack

**Pull request merged**